src/NServiceBus.AcceptanceTests/Core/BestPractices/When_publishing_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -22,7 +22,7 @@ public async Task Should_allow_publishing_commands()
             .Done(c => c.EndpointsStarted)
             .Run();
 
-        Assert.That(context.EndpointsStarted, Is.True);
+        Assert.That(context.EndpointsStarted.Task.IsCompletedSuccessfully, Is.True);
```

Comment:
Do we actually need this anymore? Would the test complete if endpoints are not started?

Reply:
I think I will improve this in a follow up